packages/block-library/src/list/edit.js

```diff
@@ -124,9 +124,14 @@ function IndentUI( { clientId } ) {
 }
 
 export default function Edit( { attributes, setAttributes, clientId, style } ) {
+	const { ordered, type, reversed, start } = attributes;
 	const blockProps = useBlockProps( {
-		...( Platform.isNative && { style } ),
```

Comment:
Not sure why this was ignored for native. Does that have any consequences for the new inline style?

Reply:
Doesn't this line mean that the Edit component's `style` prop is used _only_ on native platforms? This line appears to have originally been changed in #42711, which changed the List block to use nested blocks.

https://github.com/WordPress/gutenberg/pull/42711/files#diff-99bb126c574fb4790a60e7b6cf372c8a8bce3114f8a9740d0f92e47e74238724R133